en/04_Changelogs/5.0.0.md

```diff
@@ -23,12 +32,68 @@ title: 5.0.0 (unreleased)
   - [URLs generated by Silverstripe CMS don't include a trailing slash by default](#trailing-slash)
   - [CWP Agency Extensions](#cwp-agency-extensions)
 - [Features and enhancements](#features-and-enhancements)
-  - [Other features](#other-features)
+  - [Extension changes](#extension-changes)
+  - [Other new features](#other-features)
 - [Bugfixes](#bugfixes)
 
+## Change to supported modules {#supported-modules}
+
+Some Silverstripe CMS modules are commercially supported. Silverstripe commits to looking after those modules for the duration of the Silverstripe CMS 5 lifecycle.
+
+Review the list of [Commercially Supported Modules for Silverstripe CMS 5](/project_governance/supported_modules).
+
+### Community supported modules in Silverstripe CMS 5
+Some modules that were commercially supported in Silverstripe CMS 4 will not be supported in Silverstripe CMS 5. Some of those modules will provide CMS5-compatible versions. Others will be dropped altogether.
+
+Just because a module is not "commercially supported", doesn't mean that you shouldn't be using it. Community supported modules are maintained on a "best-effort" basis. You should take this into consideration when choosing to install a community supported module in your project.
+
+Email [community@silverstripe.org](mailto:community@silverstripe.org) if you are keen to help maintain some of the modules that will no longer be commercially supported.
+
+Module | Status | Notes
```

Comment:
Please add [dnadesign/silverstripe-elemental-subsites](https://packagist.org/packages/dnadesign/silverstripe-elemental-subsites) to this list.